scripts/docker-run.sh

```diff
@@ -10,7 +10,7 @@
 #  To build topology:
 #  ./scripts/docker-run.sh ./scripts/build-tools.sh
 
-docker run -i -t -v "$(pwd)":/home/sof/work/sof.git \
+docker run -i -v "$(pwd)":/home/sof/work/sof.git \
```

Comment:
@marc-hb Do you try to run this CMD locally?
I know this will fix CI system issues, but it will also break local run. In Jenkins we have manfully copy paste a same line to run with docker. I think it is time to have an option or another wrapper for CI systems.

Reply:
As I mentioned in the commit message, Interactive docker users can run `docker exec sof bash` and run commands interactively directly from there. There is no point in an indirection layer for interactive use.

Reply:
Do you notice that we are mapping our repo source with `-v "$(pwd)":/home/sof/work/sof.git`?
It would be not that easy for user to start interactive shell like this.

Reply:
Thanks for pushing to spend more time on docker.

> It would be not that easy for user to start interactive shell like this.

I looked for something like `docker --args-from-file` but didn't find anything.

> Do you try to run this CMD locally?

Removing the `--tty` option does not break any build but it does make `./scripts/docker-run.sh /bin/bash` less usable.

I found a more flexible way, see current #3761